src/NServiceBus.RavenDB.AcceptanceTests/Config.cs

```diff
@@ -1,24 +1,34 @@
 ﻿using System;
 using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
 using NServiceBus.Persistence;
+using NServiceBus.Settings;
 using Raven.Client.Document;
 
 public class ConfigureRavenDBPersistence
 {
     DocumentStore documentStore;
 
+    const string DefaultDocumentStoreKey = "$.ConfigureRavenDBPersistence.DefaultDocumentStore";
+    const string DefaultPersistenceExtensionsKey = "$.ConfigureRavenDBPersistence.DefaultPersistenceExtensions";
+
     public void Configure(BusConfiguration config)
     {
         documentStore = new DocumentStore
         {
             Url = "http://localhost:8083",
-            DefaultDatabase = Guid.NewGuid().ToString(),
-            ResourceManagerId = Guid.NewGuid() /* This is OK for ATT purposes */
+            DefaultDatabase = Guid.NewGuid().ToString()
         };
 
-        documentStore.Initialize();
+        var settings = config.GetSettings();
+        
+        settings.Set(DefaultDocumentStoreKey, documentStore);
```

Comment:
Because this is a unique-to-the-acceptance-tests thing I wanted to be as outside the normal address space / not stepping on any toes as possible